src/modules/graphics/Graphics.cpp

```diff
@@ -1700,6 +1700,15 @@ void Graphics::origin()
 	pixelScaleStack.back() = 1;
 }
 
+love::math::Transform *Graphics::buildTransform()
+{
+	Matrix4 &m = transformStack.back();
+
+	love::math::Transform *t = new love::math::Transform();
+	t->setMatrix(m);
```

Comment:
It's not super important but this can be one call: `love::math::Transform *t = new love::math::Transform(m);`

Reply:
Hmm, the whole thing could be `return new love::math::Transform(transformStack.back());`.

Would that be unclean?

Reply:
Yeah that'd work too, and it's a bit cleaner – but the choice between the two is mostly personal preference IMO.